src/psyclone/tests/core/symbolic_maths_test.py

```diff
@@ -240,3 +238,34 @@ def test_symbolic_math_functions_with_constants(fortran_reader, expressions):
     schedule = psyir.children[0]
     sym_maths = SymbolicMaths.get()
     assert sym_maths.equal(schedule[0].rhs, schedule[1].rhs) is True
+
+
+@pytest.mark.parametrize("expressions", [("field(1+i)", "field(i+1)"),
+                                         ("a%field(b+1)", "a%field(1+b)"),
+                                         ("a%b%c(a_b+1)", "a%b%c(1+a_b)"),
+                                         ("a%field(field+1)",
+                                          "a%field(1+field)"),
+                                         ("b+a%b(a%c,a%c,a%c)",
+                                          "b+a%b(a%c,a%c,a%c)")
+                                         ])
+def test_symbolic_math_use_reserved_names(fortran_reader, expressions):
+    '''Test that reserved names are handled as expected. The SymPy parser
+    uses 'eval' internally, so if a Fortran variable name should be the
+    same as a SymPy function (e.g. 'field'), parsing will fail.
+
+    '''
+    print("--------------------------")
+    # A dummy program to easily create the PSyIR for the
+    # expressions we need. We just take the RHS of the assignments
+    source = '''program test_prog
+                 use some_mod
+                 integer :: field(10), i
+                 type(my_mod_type) :: a, b
+                 x = {0}
+                 x = {1}
+                 end program test_prog
+             '''.format(expressions[0], expressions[1])
```

Comment:
Please use an f-string.

Reply:
Done. I think I've fixed up all modified files in this PR with f-strings.